transform/person_serializer_test.go

```diff
@@ -11,20 +11,17 @@ import (
 func TestSerializePersonResourceWithName(t *testing.T) {
 	fakeSparql := new(MockedReader)
 
-	// TODO Fix
 	indexer := NewPersonSerializer(repository.NewService(fakeSparql))
 
-	dept := "http://example.com/department1"
-	inst := "http://example.com/institution1"
 	resource := &models.Person{
 		Firstname:      "Harry",
 		Lastname:       "Potter",
 		URI:            "http://example.com/record1",
-		DepartmentURI:  &dept,
-		InstitutionURI: &inst,
+		DepartmentOrgs:  []*models.PositionOrganization{ &models.PositionOrganization{ "http://example.com/department1", "Department 1" } },
```

Comment:
can you initialize these structs with the field names (rather than positionally)? This makes it easier to understand the intention, especially if we add a field in the future.